pandas/tests/io/json/test_ujson.py

```diff
@@ -559,6 +560,16 @@ def test_encode_long_conversion(self):
         assert output == json.dumps(long_input)
         assert long_input == ujson.decode(output)
 
+    @pytest.mark.parametrize("bigNum", [sys.maxsize + 1, -(sys.maxsize + 1)])
```

Comment:
```suggestion
    @pytest.mark.parametrize("bigNum", [sys.maxsize + 1, -(sys.maxsize + 2)])
```

To exceed the negative range would have to add 2